### ISSUE TITLE
demo.py: No module named doc

### ISSUE THREAD

**Issue by ashwani99** [HTTP-APIs/hydrus#176]
### I'm submitting a 
- [x] bug report.

### Current Behaviour:
Running `demo.py` raises `ModuleNotFoundError`

### Steps to reproduce:
Run the following commands:
```bash
$ cd hydrus
$ python demo.py
Traceback (most recent call last):
  File "hydrus/demo.py", line 12, in <module>
    from doc import doc
ModuleNotFoundError: No module named 'doc'
```

### Do you want to work on this issue?
Yes

**Comment by ashwani99** [comment]
@Mec-iS  Should the files needed for executing demo.py put in a separate directory?

**Comment by Mec-iS** [comment]
Is hydrus directory in the `PYTHONPATH` of the environment you are using?

**Comment by ashwani99** [comment]
@Mec-iS Yes it is. Actually the error is happening because the location of `main.py` was changed in this [commit](https://github.com/HTTP-APIs/hydrus/commit/97308c10fde33286ac6c90e9a0e681f558c11196) and renamed to `demo.py`

**Comment by IamRaviTejaG** [comment]
@Mec-iS @ashwani99 The above PR fixes the issue. Please have a look.

**Comment by ashwani99** [comment]
Sure it fixes the issue but I think it makes the repo little unorganized. As `demo.py` is an example script it should be put inside `examples` to separate out from the actual code as it has nothing to do with the code for hydrus.

**Comment by xadahiya** [comment]
Fixed with #177
